src/sysinfo.cc

```diff
@@ -84,7 +84,7 @@
 namespace benchmark {
 namespace {
 
-void PrintImp(std::ostream& out) { out << std::endl; }
+void PrintImp(std::ostream& out) { out << '\n'; }
```

Comment:
Should we flush stream[s] before calling `std::exit()`, since it does not do that automatically?

Reply:
adding to everywhere we explicitly `exit`

Reply:
There's probably other places where we might flush intentionally, but this is better than nothing.